cddl/contrib/opensolaris/tools/ctf/cvt/ctf.c

```diff
@@ -1316,16 +1316,20 @@ resurrect_types(ctf_header_t *h, tdata_t *td, tdesc_t **tdarr, int tdsize,
 			tdp->t_fndef = xcalloc(sizeof (fndef_t));
 			tdp->t_fndef->fn_ret = tdarr[type];
 
-			v = (void *) (dptr + (idwidth * (vlen - 1)));
-			if (vlen > 0 && *(uint_t *)v == 0)
-				tdp->t_fndef->fn_vargs = 1;
+			if (vlen > 0) {
+				argid = 0;
```

Comment:
Is it possible that `idwidth` is not `sizeof(uint_t)` but smaller?  If so, then whatever "extra" bytes are in the stream before this argument can cause this to perhaps set `fn_vargs` to 1 incorrectly?

Reply:
v2 has it as a short, v3 has it as an int, see my comment.

Reply:
Looking at commit messages, the tools should always emit v3, so hopefully it doesn't matter.